addons/mrp_landed_costs/tests/test_stock_landed_costs_mrp.py

```diff
@@ -71,6 +69,7 @@ def setUpClass(cls):
         cls.landed_cost = cls.env['product.product'].create({
             'name': 'Landed Cost',
             'type': 'service',
+            'categ_id': cls.categ_all.id,
```

Comment:
same why need to set category in test case?
It is falling without it?

Reply:
:thinking:

Reply:
When adding Landed Costs in Odoo, the product category must use FIFO or AVCO costing methods. By default, products use Standard Costing, which doesn't support landed cost adjustments and will raise an error.

![2024-09-12_12-46](https://github.com/user-attachments/assets/b2c6ac3b-4895-47a8-bcf8-5bc1dba88a54)

To prevent this, we set the costing method for our category (categ_all) to FIFO:

`cls.product_refrigerator.categ_id.property_cost_method = 'fifo'`